abbr/views.py

```diff
@@ -13,10 +13,13 @@
     validate_url,
 )
 
+from ago import human
+
 
 def get_expiries():
     now = datetime.now()
     return [
+        {'name': 'default expiry (' + human(get_expiry(), precision=1) + ")", 'value': from_datetime(get_expiry()) },
```

Comment:
It might be better to use the human() method on all fields here, for consistency.